shared/js/background/settings.es6.js

```diff
@@ -38,6 +42,15 @@ function buildSettingsFromLocalStorage () {
     })
 }
 
+function buildSettingsFromManagedStorage () {
+    return new Promise((resolve) => {
```

Comment:
Why remove the `reject`?

Reply:
We are not using it in this function, eslint would complain about it if we left it.